### PR TITLE
Alibaba Cloud Container Service support

### PR DESCRIPTION
**Pull request by bonifaido**
Fixes: #775

### CODE REVIEW

File: cluster/accs.go [hunk 1]
```diff
@@ -0,0 +1,946 @@
...
+				inf.InternetURI = inter
+			}
+		}
+	}
+	if inf.JumpHost == "" {
+		err = errors.New("ssh jump host not found")
+		return
+	}
+	if inf.IntranetURI == "" {
+		err = errors.New("ssh jump host not found")
```
Comment (baluchicken) [inline cluster/accs.go]: wrong error

File: cluster/accs.go [hunk 2]
```diff
@@ -0,0 +1,946 @@
...
+		return
+	}
+	if inf.InternetURI == "" {
+		err = errors.New("ssh jump host not found")
```
Comment (baluchicken) [inline cluster/accs.go]: wrong error

File: cluster/accs.go [hunk 3]
```diff
@@ -0,0 +1,946 @@
...
+		return err
+	}
+
+	cluster, err := waitForClusterState(client, r.ClusterID)
+	if err != nil {
+		return err
+	}
+
+	updatedNodePools := make([]*model.ACCSNodePoolModel, 0, 1)
+	updatedNodePools = append(updatedNodePools, nodePoolModels[0])
```
Comment (baluchicken) [inline cluster/accs.go]: Can this be simplified ? updatedNodePools= nodePoolModels[0]

**Comment by bonifaido** [comment]
Review comments fixed. Thanks!

**Pull request merged**